pymeasure/instruments/hp/hp663X2A.py

```diff
@@ -221,3 +221,9 @@ def enable_cont_output_trigger(self):
     def write(self, command):
         """Write a command to the instrument."""
         super().write(command)
+
+class HP66312A(HP663X2A)
+    pass
+
+class HP66332A(HP663X2A)
```

Comment:
The documentation (in docs folder) will only refer to the older device and won't show this one.

Please add it to the docs (and probably the parent device).

It these two devices are the only ones of the series, and the driver won't have any differences, the HP633X2A driver will be sufficient (with an appropriate docstring).